app/src/main/res/values-de/strings.xml

```diff
@@ -90,7 +90,7 @@ Bitte melden Sie auftretende Probleme via GitHub.</string>
   <!--Setting title-->
   <string name="folder_master">Nur senden</string>
   <!--Setting title-->
-  <string name="folder_fileWatcher">Echtzeit-Synchronisation</string>
+  <string name="folder_fileWatcher">Auf Änderungen achten</string>
```

Comment:
This might be overwritten by Transifex. Better to add any translations via their website.

Reply:
Sure, I'll also post it on transifex. Just changed it so see how it fits on my 5,5'' screen.